src/java.base/share/classes/java/util/regex/MatchResult.java

```diff
@@ -76,6 +80,43 @@ public interface MatchResult {
      */
     public int start(int group);
 
+    /**
```

Comment:
Is there a reason why you added this code here? In the JDK 20 sources it is after the old `group` functions.
It might make merging in the future easier if it is in the same spot.